react/components/AdvancedNotificationBar/index.tsx

```diff
@@ -84,53 +113,71 @@ function AdvancedNotificationBar({
 
   return (
     <div
-      className={`${handles.notificationBarContainer} relative w-100 pv5`}
+      className={`${handles.notificationBarContainer}${
+        blockClass && `--${blockClass}`
+      } relative w-100 pv5`}
```

Comment:
nu inteleg sintaxa asta de className. 
care este scopul?

Reply:
Am schimbat aici. Am un parametru din setari "blockClass" care poate fi setat din site editor pe fiecare notification bar din lista individual. Si acesta va fi pus in rootul componentei pentru a putea fi selectat un bar individual din store them, cum a cerut daniel